content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,320 @@
+
+Once the account is created, a confirmation email is sent to the email used to create the account,
+containing the account credentials to configure the account directly in the Uruware portal:
+
+- Testing portal: https://odootest.ucfe.com.uy/Gestion/
+- Production portal: https://prod6109.ucfe.com.uy/Gestion/
+
+Use the account credentials in the email to log in to the portal.
+
+Electronic invoice data
+-----------------------
+
+To configure the electronic invoice data, an environment and credentials need to be configured. To
+do so, navigate to :menuselection:`... --> ...`.
+
+First, select the :guilabel:`UCFE Web Services` environment:
+
+- :guilabel:`Production`: for production databases. In this mode, electronic documents are sent to
+  |DGI| through Uruware for their validation.
+- :guilabel:`Testing`: for test databases. In this mode, the direct connection flows can be tested,
+  with the files sent to the |DGI| testing environment through Uruware.
+- :guilabel:`Demo`: files are created and accepted automatically in demo mode but are **not** sent
+  to the |DGI|. For this reason, rejection errors will not appear in this mode. Every internal
+  validation can be tested in demo mode. Avoid selecting this option in a production database.
+
+Then, enter the :guilabel:`Uruware Credentials`:
+
+- :guilabel:`Uruware WS Password`
+- :guilabel:`Commerce Code`
+- :guilabel:`Terminal Code`
+
+.. image:: uruguay/electronic-invoice-data.png
+   :alt: Required information for electronic invoice.
+   :align: center
+
+.. note::
+   These credentials can be obtained from the email received when the :ref:`Uruguay account was
+   created <l10n_uy/uruware-account>`.
+
+Configure master data
+---------------------
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+The :doc:`chart of accounts <../accounting/get_started/chart_of_accounts>` is installed by default
+as part of the set of data included in the localization module, the accounts are mapped
+automatically in taxes, default accounts payable, and default accounts receivable.
+
+Accounts can by added or deleted according to the company's needs.
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+Contacts
+~~~~~~~~
+
+To create a contact, navigate to :menuselection:`Contacts app` and select :guilabel:`New`. Then
+enter the following information:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`: required to confirm an electronic invoice.
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`: required to confirm an electronic invoice.
+
+- :guilabel:`Identification Number`:
+
+  - :guilabel:`Type`: select a identification number type.
+  - :guilabel:`Number`: required to confirm an electronic invoice.
+
+Review taxes
+~~~~~~~~~~~~
+
+As part of the Uruguay localization module, taxes are automatically created with its configuration
+and related financial accounts.
```

Comment:
Need navigation instructions to where taxes can be reviewed.

Reply:
Add the following:

To review these taxes, go to :menuselection:`Accounting --> Configurations --> Taxes`.